frontend/src/store/store.js

```diff
@@ -42,87 +44,44 @@ export const store = new Vuex.Store({
         state.sizeGB = 'big'
       }
     },
-    loginfail: (state, loginfail) => {
-      //console.log('mutation: email')
-      state.loginfail = loginfail
-    },
     email: (state, email) => {
-      //console.log('mutation: email')
       state.email = email
     },
     session_id: (state, session_id) => {
-      //console.log('mutation: session_id')
       state.session_id = session_id
     },
     user_balance: (state, balance) => {
-      //console.log('mutation: user_balance')
       state.user.balance = balance / 10000
     },
     user_balance_gdt: (state, balance) => {
-      //console.log('mutation: user_balance_gdt')
       state.user.balance_gdt = balance / 10000
     },
   },
   // Asyncronous actions - used for api calls
   actions: {
     login: async ({ dispatch, commit }, data) => {
-      const result = await loginAPI.login(data.email, data.password)
-      if (result.success) {
-        commit('session_id', result.result.data.session_id)
-        commit('email', data.email)
-        $cookies.set('gdd_session_id', result.result.data.session_id)
-        $cookies.set('gdd_u', data.email)
-        router.push('/overview')
-      } else {
-        // Register failed, we perform a logout
-        //alert('>>>>> FAIl LOGIN')
-        commit('loginfail', true)
-
-        //dispatch('logout')
-      }
-    },
-    passwordReset: async (data) => {
-      //console.log('<<<<<<<<<<< PASSWORT RESET TODO >>>>>>>>>>>', data.email)
+      commit('session_id', data.session_id)
+      commit('email', data.email)
     },
+    passwordReset: async (data) => {},
     schoepfen: async (data) => {
       // http://localhost/transaction-creations/ajaxCreate
     },
     createUser: async ({ commit, dispatch }, data) => {
-      // console.log('action: createUser')
-      const result = await loginAPI.create(
-        data.email,
-        data.first_name,
-        data.last_name,
-        data.password,
-      )
-      if (result.success) {
-        commit('session_id', result.result.data.session_id)
-        commit('email', data.email)
-        $cookies.set('gdd_session_id', result.result.data.session_id)
-        $cookies.set('gdd_u', data.email)
-        router.push('/overview')
-      } else {
-        // Register failed, we perform a logout
-        // console.log('action createUser to  logout start')
-        dispatch('logout')
-      }
+      commit('session_id', data.session_id)
+      commit('email', data.email)
     },
     logout: async ({ commit, state }) => {
-      //console.log('action: logout')
       if (state.session_id) {
```

Comment:
Make this call in the component?

Reply:
I leave this as it is by now. When we switch to one menu I will move the call to the component